src/components/common/layout/base.njk

```diff
@@ -104,6 +104,7 @@
 {% endblock %}
 
 {% block bodyEnd %}
-    <script type="text/javascript" src="/public/scripts/cookies.js"></script>
-    <script type="text/javascript" src="/public/scripts/application.js"></script>
+  <script type="text/javascript" src="/public/scripts/cookies.js"></script>
+  <script type="text/javascript"  src="/public/scripts/application.js"></script>
+  <script type="text/javascript" nonce='{{scriptNonce}}'>window.GOVSignIn.appInit("{{gtmId}}")</script>
```

Comment:
This will be in every page regardless of whether analytics consent has been given.  Is that ok?

Reply:
yes